scripts/unit-typescript-esm.js

```diff
@@ -4,9 +4,9 @@ const { exec } = require('node:child_process')
 
 const args = [
   'tap',
-  '--node-arg=--loader=ts-node/esm',
+  '--node-arg=--import=ts-node/esm',
```

Comment:
Are you sure `--import=ts-node/esm` will works?

Reply:
It works because `tap` handle the transpile, you can see it run even if you remove `--node-arg=--import=ts-node/esm`